scripts/gold.js

```diff
@@ -81,10 +81,10 @@ module.exports = robot => {
     if (!now) now = new Date()
     const expire = new Date(now.getTime() + diff)
     goldUsers[name] = { user: name, expire: expire }
-    let message = ':clap2: eres miembro golden :monea: por 1 mes!'
+    let message = ':clap2: eres miembro gold :monea: por 1 mes!'
     if (key === null) {
       channelId = robot.adapter.client.rtm.dataStore.getChannelByName(process.env.GOLD_CHANNEL || '#random').id
-      message = `:clap2: *${name}* donó 1 mes de servidor a :huemul:, se lleva 3 stickers :huemul: y es miembro golden :monea: por 2 meses!`
+      message = `:clap2: *${name}* donó 1 mes de servidor a :huemul:, se lleva swags :devschile: y es miembro gold :monea: por 2 meses!`
```

Comment:
```suggestion
      message = `:clap2: *${name}* donó 1 mes de servidor a :huemul:, se lleva swag :devschile: y es miembro gold :monea: por 2 meses!`
```

Creo que es mejor decir swag en singular.